packages/domain-picker/jest.config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
 	preset: '../../test/packages/jest-preset.js',
 	testEnvironment: 'jsdom',
+	testMatch: [ '<rootDir>/**/__test__/**/*.[jt]s?(x)', '!**/.eslintrc.*' ],
```

Comment:
Override default Calypso's config, so that we can put our tests in folders named `__test__`

Reply:
Maybe a bit of [nitpicking/] since Calypso is overruling the default Jest config anyway, but Jest's default is `__tests__` (plural) (see: https://jestjs.io/docs/configuration#testmatch-arraystring). The same goes for `__mock__` vs. `__mocks__`.

WDYT? Do we need to use plural too?

Reply:
Done across these commits:
- https://github.com/Automattic/wp-calypso/pull/51235/commits/cd951f36e0d91f5c4daea4f8571c9c81caf950f5
- https://github.com/Automattic/wp-calypso/pull/51235/commits/a5c22b64f101483b8e32811a250a4865bed11d47
- https://github.com/Automattic/wp-calypso/pull/51235/commits/eea4c35d2ebe964020cbecdd29e5759b8566563a
- https://github.com/Automattic/wp-calypso/pull/51235/commits/512ee17ce81e4ea1c82ab32dd4b95fd2d59e108d